### PR TITLE
Send DestroyEntitiesMessage when entity is removed

### PR DESCRIPTION
**Pull request by Paulomart**
Fix for #170

### CODE REVIEW

File: src/main/java/net/glowstone/entity/GlowEntity.java [hunk 1]
```diff
@@ -695,6 +696,10 @@ public void remove() {
         active = false;
         world.getEntityManager().unregister(this);
         server.getEntityIdManager().deallocate(this);
+        DestroyEntitiesMessage message = new DestroyEntitiesMessage(Arrays.asList(id));
+        for (GlowPlayer player : getWorld().getRawPlayers()) {
+            player.getSession().send(message);
```
Comment (mastercoms) [inline src/main/java/net/glowstone/entity/GlowEntity.java]: Check if player can see?

Comment (Paulomart) [inline src/main/java/net/glowstone/entity/GlowEntity.java]: oh sure

**Comment by kamcio96** [comment]
It isn't good solution

**Comment by mastercoms** [comment]
Why not?

**Comment by kamcio96** [comment]
Eg plugin can call remove few times.
It should be do here: https://github.com/GlowstonePlusPlus/GlowstonePlusPlus/blob/master/src/main/java/net/glowstone/entity/GlowPlayer.java#L508

**Comment by Paulomart** [comment]
Was searching for this, will update pr

**Comment by kamcio96** [comment]
You should work around that method, you can override it or change https://github.com/GlowstonePlusPlus/GlowstonePlusPlus/blob/master/src/main/java/net/glowstone/entity/GlowPlayer.java#L511

**Pull request merged**